sdk/template/template/test/snippets.spec.ts

```diff
@@ -0,0 +1,29 @@
+import { DefaultAzureCredential } from "@azure/identity";
+import { setLogLevel } from "@azure/logger";
+import { ConfigurationClient } from "../src";
+
+describe("snippets", function () {
+  it("new_configurationclient", function () {
+    // @ts-ignore
+    const client = new ConfigurationClient(
+      process.env.ENDPOINT ?? "<app configuration endpoint>",
+      new DefaultAzureCredential()
+    );
+  });
+
+  it("FooBar", () => {
```

Comment:
Should we keep these as markers for testing purposes or should we have samples strictly for `template`?